pac4j-saml/src/main/java/org/pac4j/saml/store/HttpSessionStore.java

```diff
@@ -34,7 +37,7 @@
     /**
      * Internal store for messages, corresponding to the object in session.
      */
-    private LinkedHashMap<String, XMLObject> internalMessages;
+    private LinkedHashMap<String, String> internalMessages;
```

Comment:
I realize this has always been here, but is there a reason LinkedHashMap is required? Could we just use the usual HashMap instead?

Reply:
Good question!

Reply:
I guess that there might be some performance idea behind that: messages are saved and should be retrieved in order...

Reply:
thanks